app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -21,160 +15,106 @@ import androidx.compose.foundation.layout.Spacer
-
-                Placeholder {
-                    Box(
-                        Modifier.background(
-                            when (isSystemInDarkTheme()) {
-                                true -> Color(0xFF42414D) // DarkGrey30
-                                false -> PhotonColors.LightGrey30
-                            }
-                        )
-                    )
-                }
-
-                Fallback {
-                    Box(
-                        Modifier.background(
-                            when (isSystemInDarkTheme()) {
-                                true -> Color(0xFF42414D) // DarkGrey30
-                                false -> PhotonColors.LightGrey30
-                            }
-                        )
-                    )
-                }
-            }
-        }
-
-        Spacer(modifier = Modifier.height(8.dp))
-
-        CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
-            Text(
-                modifier = Modifier.padding(bottom = 2.dp),
-                text = story.publisher,
-                style = MaterialTheme.typography.caption,
-                maxLines = 1,
-                overflow = TextOverflow.Ellipsis
-            )
+    val imageUrl = story.imageUrl.replace(
+        "{wh}",
+        with(LocalDensity.current) { "${116.dp.toPx().roundToInt()}x${84.dp.toPx().roundToInt()}" }
+    )
+    ListItemTabLarge(
+        client = client,
+        imageUrl = imageUrl,
+        onClick = { onStoryClick(story) },
+        title = { TabTitle(text = story.title, maxLines = 3) },
+        subtitle = {
+            TabSubtitleWithInterdot(story.publisher, "${story.timeToRead} min")
         }
-        Text(
-            text = story.title,
-            style = MaterialTheme.typography.subtitle1,
-            maxLines = 4,
-            overflow = TextOverflow.Ellipsis
-        )
-    }
+    )
 }
 
 /**
- * Displays a list of [PocketRecommendedStory]es.
+ * Displays a list of [PocketRecommendedStory]es on 3 by 3 grid.
+ * If there aren't enough stories to fill all columns placeholdera containing an external link
+ * to go to Pocket for more recommendations are added.
+ *
+ * @param stories the list of [PocketRecommendedStory]ies to be displayed. Expect a list with 8 items.
+ * @param client [Client] instance to be used for downloading the story header image.
+ * @param onExternalLinkTapped callback for when the user taps an element which contains an
+ * external link for where user can go for more recommendations.
  */
 @Composable
 fun PocketStories(
     @PreviewParameter(PocketStoryProvider::class) stories: List<PocketRecommendedStory>,
-    client: Client
+    client: Client,
+    onExternalLinkTapped: (String) -> Unit
 ) {
-    // Items will be shown on two rows. Ceil the divide result to show more items on the top row.
-    val halfStoriesIndex = (stories.size + 1) / 2
+    // Show stories in a 3 by 3 grid
+    val gridLength = 3
 
     LazyRow {
```

Comment:
Lazy lists automatically clip children and their shadows on the main axis - https://issuetracker.google.com/issues/186318448
This means here that children's horizontal shadows will be clipped.
Tried using a scrollbable Row. Same result.
The only solution I see involves refactoring the homescreen to remove `the 16dp padding` and let each child handle this on it's own. This way we could ensure we leave enough space for the shadows.
But right now this is the only homescreen that would need this so maybe we can wait a bit more until all homescreen views are migrated to composables and then we can have a cohesive approach.